AndroidSDKTests/src/test/java/com/leanplum/internal/RequestTest.java

```diff
@@ -67,6 +68,20 @@ public void setUp() {
     Leanplum.setApplicationContext(context);
   }
 
+  /**
+   * Test that read writes happened sequentially when calling sendNow().
+   */
+  @Test
+  public void testSequenceOfDatabaseOperations() {
+    Map<String, Object> params = new HashMap<>();
```

Comment:
test style: tests are easier to understand and maintain when documented using the Given/When/Then convention (https://www.agilealliance.org/glossary/gwt/):

for this test:

// Given a request.

// When the request is sent.

// Then the request is properly first written to the local db, and then sent.

Reply:
done